### ISSUE TITLE
Top menu not visible on mobile when logged in

### ISSUE THREAD

**Issue by dboyer** [Iridescent-CM/technovation-app#3216]
On small screens and mobile screens, the limited screen size prevents the top menu from displaying. This prevents users from logging out, seeing the survey, or accessing the link to the curriculum.

![Screenshot_20220216-151321.png](https://images.zenhubusercontent.com/60527169cde2772242d0de95/d79dd860-0c78-4394-be21-98c6138191fa)![Screen Shot 2022-02-16 at 3.15.56 PM.png](https://images.zenhubusercontent.com/60527169cde2772242d0de95/8f6e83ee-12d7-45c7-b5d5-0e8dc5bd2aeb)

**Comment by viviancan** [comment]
The bug was due to a media query and the tw-hidden class name. The navbar was being hidden on anything smaller than a tablet.

I updated the CSS and added the curriculum and about links to the dropdown menu. So now in any screens smaller than tablet, the curriculum and about links on the main navbar are hidden and show up in the dropdown. 

![Screen Shot 2022-02-18 at 4.28.36 PM.png](https://images.zenhubusercontent.com/606b0f4fd13d53dff3d14fa5/08053c93-4432-45ac-9d88-555ef01060cf)
